backend/src/repository/dalPost.ts

```diff
@@ -23,6 +23,15 @@ export const getByBoard = async (boardID: string) => {
   }
 };
 
+export const getBoardTypePosts = async (boardID: string) => {
```

Comment:
Instead of creating a new function for querying boards by type, can we update the current `getByBoard` to take an optional parameter `type?: PostType` and if it's specified by the caller, we just include it in the query?

Reply:
After this fix, feel free to merge!